### ISSUE TITLE
Incorrect warning: Bicycle parking should be a closed area

### ISSUE THREAD

**Issue by owiecc** [openstreetmap/iD#8515]
When tagging `amenity=bicycle_parking` as a way (which is allowed, see [bicycle_parking wiki](https://wiki.openstreetmap.org/wiki/Key:bicycle_parking)) iD displays an incorrect warning. 

<img width="361" alt="Screenshot 2021-05-24 at 21 47 44" src="https://user-images.githubusercontent.com/6896639/119401893-6cda3980-bcdc-11eb-9a01-3ca11f037ad2.png">

**Comment by HttTavares** [comment]
Do you think this problem can be tackled by a beginner? If so, I could get on it.

**Comment by owiecc** [comment]
That is a question for maintainers. I don't even know how the code looks. But by all means have a go at it. That is the beauty of open source. Maybe it is a super easy fix.

**Comment by matkoniecz** [comment]
"should be a closed way" is true, in all cases mapping as an area is preferable 

disclaimer: I implemented https://github.com/gravitystorm/openstreetmap-carto/pull/1364 - not rendering ways was deliberate

**Comment by sevku** [comment]
> "should be a closed way" is true, in all cases mapping as an area is preferable
> 
> disclaimer: I implemented [gravitystorm/openstreetmap-carto#1364](https://github.com/gravitystorm/openstreetmap-carto/pull/1364) - not rendering ways was deliberate

I think sometimes it can be useful to map it as a line.

**Comment by Discostu36** [comment]
I don't think this is a discussion that should be happening here. [The wiki documentation allows bicycle parking as a way for 12 years now](https://wiki.openstreetmap.org/w/index.php?title=Tag:amenity%3Dbicycle_parking&diff=prev&oldid=643718). The reasoning is that there might also be bicycle stands that don't cover an area but are only linear (for example, one long bar for multiple bicycles). If someone thinks this is a mistake, this can be discussed on the wiki or changed with a proposal. **The ID editor validator should not check if the tags are matching the preferred mapping style of its maintainers, but if they match the documented community consensus.**

**Comment by joshinils** [comment]
Related; https://github.com/gravitystorm/openstreetmap-carto/issues/2711

**Comment by Discostu36** [comment]
It was brought to my attention that my last comment has a harsh, maybe accusing tone to it. I am sorry about that. But it is very important to me that ID and carto, being presented so prominently at osm.org that they are practically the offical editor and map style of OSM, do reflect community consensus. I do not think that the small number of people contributing to these projects should use their powerful position to establish tagging practices that they find useful, while all other OSM members would need to write a proposal or find other ways to convince the majority of the community for that.

For that reason I think that the ID validator should always only give warnings in clear cases that are not a matter of opinion.

**Comment by matkoniecz** [comment]
>  I do not think that the small number of people contributing to these projects

Note that also applies to OSM Wiki (does not affect this specific case, but "must always follow whatever is stated at OSM Wiki" is not a good rule either - often it is OSM Wiki that needs to be changed)

**Comment by Discostu36** [comment]
>> I do not think that the small number of people contributing to these projects
> Note that also applies to OSM Wiki (does not affect this specific case, but "must always follow whatever is stated at OSM Wiki" is not a good rule either - often it is OSM Wiki that needs to be changed).

Yes, the OSM Wiki should, but does not always reflect the community consensus. But I think the right order would be to first edit the wiki (after discussion), then add the validation rule.

> Oh, and in addition "small number of people" applies also to participation in tagging discussions.

Well, yes, but I hope that by including the community forum into the Proposal process, the number of people taking part in it will increase. I also don't think that a validator should/must include everything that was decided in a tagging discussion or proposal. But if a rule is written for it, then it should be uncontroversial.

**Comment by matkoniecz** [comment]
oh, for an additional context: 

(1) when I was implementing OSM Carto rule then it was less clear and I admit nowadays it is clear that line tagging has support from community.

(2) this warning is semi-automatic - noone added rule specifically for this case, but it is in presets as node/area and therefore line rendering is detected automatically as supsicious
